src/libecalc/core/models/compressor/sampled/compressor_model_sampled.py

```diff
@@ -253,6 +254,18 @@ def evaluate_rate_ps_pd(
 
         return result
 
+    def evaluate_streams(
+        self,
+        inlet_streams: List[Stream],
+        outlet_stream: Stream,
+    ) -> CompressorTrainResult:
+        mixed_input_streams = Stream.mix_all(streams=inlet_streams)
```

Comment:
I think it makes sense to mix here, if we remove the one we already had, to not do it twice for different usage. @olelod what do you think?